notification/notification.go

```diff
@@ -41,8 +42,15 @@ func NewWorkItemCreated(workitemID string) Message {
 }
 
 // NewWorkItemUpdated creates a new message instance for the updated WorkItemID
-func NewWorkItemUpdated(workitemID string) Message {
-	return Message{MessageID: uuid.NewV4(), MessageType: "workitem.update", TargetID: workitemID}
+func NewWorkItemUpdated(workitemID string, revisionID uuid.UUID) Message {
+	return Message{
+		MessageID:   uuid.NewV4(),
+		MessageType: "workitem.update",
+		TargetID:    workitemID,
+		Custom: map[string]interface{}{
```

Comment:
Why do you pass `revisionID` in a map?

Reply:
Possibly be cause the notification system already accepts a "custom" map.

Reply:
I am passing it in a map because that is how the notification service will accept it.